res/styles/home.css

```diff
@@ -1,9 +1,10 @@
-html body {
+html, body {
     background: white;
-
     transition: ease 2s background;
     padding: 0;
     margin: 0;
+    max-width: 100%;
+    overflow-x: hidden;
```

Comment:
I guess don't hide the overflow, instead just try to restrict content within the window width. Hiding overflow could result in hiding actual content in some cases.